hydro_deploy/core/src/rust_crate/mod.rs

```diff
@@ -194,30 +194,3 @@ impl ServiceBuilder for RustCrate {
         )
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::deployment;
-
-    #[tokio::test]
-    async fn test_crate_panic() {
-        let mut deployment = deployment::Deployment::new();
-
-        let service = deployment.add_service(
```

Comment:
Could we bring this back? It's a good quick test and we hit this code path via Hydro framework.